java/core/src/main/java/co/worklytics/psoxy/gateway/impl/oauth/AccessTokenDto.java

```diff
@@ -0,0 +1,30 @@
+package co.worklytics.psoxy.gateway.impl.oauth;
+
+import com.google.auth.oauth2.AccessToken;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+import java.util.Optional;
+
+/**
+ * Custom class that maps 1:1 with {@link com.google.auth.oauth2.AccessToken}
+ * Used for storage of the token in JSON. Google's is immutable and can't easily configure
+ * Jackson to do it.
+ */
+@NoArgsConstructor //for jackson
+@AllArgsConstructor
+@Getter
+public class AccessTokenDto {
```

Comment:
`Serializable`? `@Data`?

Reply:
Made in immutable. No need to be serializable as we do json back and forth